packages/default-theme/src/components/account/SwNewsletter.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="newsletter-subcription">
+    <slot name="message">
+      <p class="message">
+        {{
+          $t("Here you can change your newsletter status")
+        }}
+      </p>
+    </slot>
+    <SwInput
+        v-model="user.email"
```

Comment:
we shouldn't modify the user's email here, for form purposes we can add prefilled ref, for example
```ts
const userEmail = ref(user.value?.email);
```